lambda/src/main/scala/pricemigrationengine/services/CohortTableLive.scala

```diff
@@ -44,23 +64,62 @@ object CohortTableLive {
       AttributeAction.PUT
     )
 
+  private def instantFieldUpdate(fieldName: String, instant: Instant) =
+    fieldName -> new AttributeValueUpdate(
+      new AttributeValue().withS(DateTimeFormatter.ISO_DATE_TIME.format(instant.atZone(ZoneOffset.UTC))),
```

Comment:
[Instant.now](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#now--) gives the current UTC time, but could also use the ZIO `clock` as part of the environment as you're already using it.

Reply:
The instant does ultimately come from the ZIO.clock it gets passed in here via the cases class that is being serialised?

Reply:
Ignore!